### PR TITLE
@skylar/ci test reporting

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

File: e2e/logs/test-info.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "test-name":"Detox e2e - iOS Results" }
```
Comment (brunobar79) [inline e2e/logs/test-info.json:1]: This file is duplicated in different directories, so I assume we only need one.

**Review by brunobar79 — APPROVED** [review]
This is great! Just one minor comment.

Also not sure if this is  a bug on bitrise end but the test execution times are off by a lot.

Total Time for the e2e suite: 

Screen Shot 2020-10-12 at 1 14 30 PM


Time per test based on reporting:
Screen Shot 2020-10-12 at 1 13 47 PM

**Review by osdnk — APPROVED** [review]
I would merge!

**Review by osdnk — APPROVED** [review]
I would merge!

**Pull request merged**